[Bug] Build scripts failing

### Current Behavior

Currently, when trying to perform a PR, the build scripts fail to run, due to trying to run `npm run test:ci` which has not been set up.

### Relevant log output

```shell
Run npm run test:ci
  npm run test:ci
  cat ./coverage/lcov.info
  shell: /usr/bin/bash -e {0}
  env:
    CI: true
npm ERR! Missing script: "test:ci"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/2023-08-16T22_40_58_689Z-debug-0.log
Error: Process completed with exit code 1.
```


### Expected Behavior

Build script should complete.

### Changes Needed

Remove the `npm run test:ci`from GitHub actions file.

### Location of Issue

.github/workflows/build.yml

### Code Snippets

_No response_

### Additional Context

_No response_

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -26,23 +26,17 @@ jobs:
         with:
           persist-credentials: false
 
-      - name: 🔢 Use Node.js ${{ matrix.node-version }}
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.4.0
+
+      - name: 🔢 Use Node.js version 18
         uses: actions/setup-node@v3
         with:
-          node-version: latest
+          node-version: 18
+          cache: "pnpm"
 
-      - name: 📶 Install
-        run: |
-          npm install
-        env:
-          CI: true
-
-      - name: 🧪 Test
-        run: |
-          npm run test:ci
-          cat ./coverage/lcov.info
-        env:
-          CI: true
+      - name: 📶 Install dependencies
+        run: pnpm install
 
       # - name: 🧪 Test
       #   run: |
@@ -56,10 +50,13 @@ jobs:
       #     path: coverage/lcov.info
 
       - name: 🔧 Build
-        run: |
-          NODE_ENV=production npm run build
-        env:
-          CI: true
+        run: NODE_ENV=production npm run build
+
+      - name: Lint
+        run: pnpm run lint
+
+      - name: Check formatting
+        run: pnpm prettier --check .
 
       - name: 🔊 Save PR number (for SonarCloud)
         if: github.event_name == 'pull_request'

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -44,11 +44,16 @@ jobs:
         env:
           CI: true
 
-      - name: 🔊 Upload code coverage to GitHub Artifacts (for SonarCloud)
-        uses: actions/upload-artifact@v3
-        with:
-          name: typefaceoff-lcov-file
-          path: coverage/lcov.info
+      # - name: 🧪 Test
+      #   run: |
+      #     npm run test:ci
+      #     cat ./coverage/lcov.info
+      #
+      # - name: 🔊 Upload code coverage to GitHub Artifacts (for SonarCloud)
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: typefaceoff-lcov-file
+      #     path: coverage/lcov.info
 
       - name: 🔧 Build
         run: |

